src/content/1.7/development/components/grid/filter-types-reference/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Filter types reference
+menuTitle: Filter types reference
+weight: 13
+---
+
+## Filter types reference
+
+Filter types are actually Symfony Form types used in the grid to filter columns.
+
+### SearchAndResetType
+
+Usually inserted at last this button submit the POST filtering request, it then display a reset button to clear all the filters.
+
+| Properties         | Expected value(s)                  |
+|--------------------| -----------------------------------|
+| **Options**        | `reset_route`                      |
+|                    | `reset_route_params`               |
+|                    | `redirect_route`                   |
+|                    | `redirect_route_params`            |
+| **Defaults**       | `reset_route` => null              |
+|                    | `reset_route_params` => []         |
+|                    | `redirect_route` => null           |
+|                    | `redirect_route_params` => []      |
+| **Allowed Types**  | `reset_route` (string, null)       |
+|                    | `reset_route_params` => []         |
+|                    | `redirect_route` => (string, null) |
+|                    | `redirect_route_params` => []      |
+
+### CountryChoiceType
+
+Builds a country selector, it contains various options but they are automatically filled with the list of country.
+
+| Properties         | Expected value(s)             |
+|--------------------| ------------------------------|
+| **Options**        | `choices`                     |
+|                    | `choice_attr`                 |
+|                    | `withDniAttr`                 |
```

Comment:
This shouldn't use camel case notation The PR to fix this in the core is here https://github.com/PrestaShop/PrestaShop/pull/18686
And the doc must be updated accordingly
Thanks @PierreRambaud for spotting this issue